starter-core/src/test/groovy/io/micronaut/starter/feature/build/MicronautAotBuildPluginSpec.groovy

```diff
@@ -0,0 +1,35 @@
+package io.micronaut.starter.feature.build
+
+import io.micronaut.starter.ApplicationContextSpec
+import io.micronaut.starter.BuildBuilder
+import io.micronaut.starter.application.ApplicationType
+import io.micronaut.starter.fixture.CommandOutputFixture
+import io.micronaut.starter.options.BuildTool
+import io.micronaut.starter.options.Language
+import spock.lang.Unroll
+
+class MicronautAotBuildPluginSpec extends ApplicationContextSpec implements CommandOutputFixture {
+
+    @Unroll
+    void 'function with gradle and feature micronaut-aot for language=#language'() {
+        when:
+        String build = new BuildBuilder(beanContext, BuildTool.GRADLE)
+                .language(language)
+                .applicationType(ApplicationType.DEFAULT)
+                .features(['micronaut-aot'])
+                .render()
+
+        then:
+        build.contains('aot {')
```

Comment:
:x: Should also probably test that the AOT plugin is added.

Reply:
I have added it here: https://github.com/micronaut-projects/micronaut-starter/pull/1043/commits/cacc08c1b81444b93128fe1ba94a8d588ea622dd